debian/copyright

```diff
@@ -0,0 +1,620 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
avTranscoder is not GPL:
https://github.com/avTranscoder/avTranscoder/blob/develop/LICENSE.LGPL